src/dataDisplay/Icon/index.tsx

```diff
@@ -116,6 +118,7 @@ const icons = {
   edit,
   error,
   eth,
+  exportImg,
```

Comment:
Could we rename this to just: "export"?

Reply:
Import and export are reserved words in js. We could think of giving them another name.

Reply:
Sure! oops. Let's leave it as it is.